### DISCUSSION TITLE
FPS floor option

### DISCUSSION THREAD

**Discussion by Gaeus** [LizardByte/community#131] [Sunshine] [answered]
Hello! I'm occasionally using moonlight for remote working, and while the current feature of only sending the frames when refreshed is great for straight-up gaming, it makes desktop usage... difficult, would it be possible to add an option to add a minimum (or "floor") number of frame sent from sunshine to moonlight?

**Comment by ReenigneArcher** [comment]
What version are you using?

**Comment by Gaeus** [comment]
Hello, I'm using the v16 release for now.Le 9 janv. 2023 17:16, ReenigneArcher ***@***.***> a écrit :
What version are you using?

**Reply by ReenigneArcher** [reply]
I don't think this issue exists in v0.17.0.

**Comment by Gaeus** [comment]
Oh !

Cool, sadly I won’t be able to update until Friday (on a business trip), but I will update once… well, updated.
De : ***@***.***>
Envoyé le :lundi 9 janvier 2023 18:56
À : ***@***.***>
Cc : ***@***.***>; ***@***.***>
Objet :Re: [LizardByte/Sunshine] FPS floor option (Discussion LizardByte/Sunshine#739)


I don't think this issue exists in v0.17.0.

**Comment by famus484** [comment]
Such a feature would be very useful to me too. In some games (ex, in menus), sunshine may halve framerate, which I can definitely notice when I go back in. Using 0.18.

I'd understand if you don't have enough time though

**Comment by ReenigneArcher** [comment, marked as answer]
This has been in Sunshine for quite a while now, so closing.
